components/Slider/src/MDCSlider.m

```diff
@@ -593,7 +593,7 @@ - (CGFloat)accessibilityAdjustmentAmount {
   CGFloat adjustmentAmount = kSliderAccessibilityIncrement * range;
   if (self.numberOfDiscreteValues > 1) {
     CGFloat discreteAdjustmentAmount = range / (self.numberOfDiscreteValues - 1);
-    if (self.discrete) {
+    if (self.isDiscrete) {
```

Comment:
This would have been `[self isDiscrete]` since the getter is a method and [only properties should be accessed with dot syntax](http://google.github.io/styleguide/objcguide.html#objective-c-method-names).